tests/test_order_registry.py

```
@@ -0,0 +1,733 @@
+def test_orphan_adoption_and_unattributed(registry: OrderRegistry):
+    """Test 6: Orphan venue order is adopted, and an unmatchable one is recorded as unattributed."""
+    now_ms = 1723840000000
+
+    # 1. Pending local order waiting for adoption
+    local_pending_id = str(uuid.uuid4())
+    registry.create_order(
+        OrderRecord(
+            id=local_pending_id,
+            order_id=None,
+            condition_id="0xcond_adopt",
+            token_id="0xtok_adopt",
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "token_id"

(S106)